### PR TITLE
Collapsible SingleMetadataResult, w/ tests

### LINKED ISSUE #180: Add the ability to expand/minimize metadata entries
The metadata component should have an attribute that lets the parent control whether or not it should be expanded or not.

### PR DESCRIPTION
**Pull request by eliasvagan**
Closes #180 

As a bonus, this heavily reduces the load of all search pages

### CODE REVIEW

**Comment by AndreasJJ** [comment]
I havne't looked at the new shadow yet, but it's actually the same shadow you used on the div of the category view and municipality view

**Comment by eliasvagan** [comment]
> 
> 
> I havne't looked at the new shadow yet, but it's actually the same shadow you used on the div of the category view and municipality view

Yes, but those views are white on top of a darker background. Search results are white on white, so the top becomes invisible with too small shadow. I also made it brighter this time

**Comment by AndreasJJ** [comment]
I guess it looks a _tad less_ tacky. But you need to look over all your code as it's some pretty drastic changes that git merge conflicts don't show f.ex. how you reintroduced the "Missing feedback article" that got removed in the latest commit to development.

**Comment by AndreasJJ** [comment]
Skjermbilde 2020-03-16 kl  15 52 23
Also, how did you manage to make the padding/margin over the "release status" different between the two datasets?

**Comment by eliasvagan** [comment]
Wait what

**Comment by eliasvagan** [comment]
That does not look like the last commit. 
image

**Comment by AndreasJJ** [comment]
weird, guess ill have to check out why git isnt giving me the latest updates

File: frontend/src/sharedComponents/Metadata/ContentComps.jsx [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+    experiencePostGuid: PropTypes.string,
+  }).isRequired,
+  showCategory: PropTypes.bool,
+  showMunicipality: PropTypes.bool,
+};
+ContentExpanded.defaultProps = {
+  showCategory: false,
+  showMunicipality: false,
+};
+ContentCollapsed.propTypes = {
```
Comment (AndreasJJ) [inline frontend/src/sharedComponents/Metadata/ContentComps.jsx]: There are two propTypes here now probably from the merging. Also experiencePostGuid doesn't exist, its replaced with
```
  experiencePosts: PropTypes.arrayOf(PropTypes.shape({
    experiencePostUuid: PropTypes.string.isRequired,
    title: PropTypes.string.isRequired,
  })),
```

Comment (AndreasJJ) [reply frontend/src/sharedComponents/Metadata/ContentComps.jsx]: Nvm, I can't read, but the experiencePostGuid still doesnt exist anymore

**Pull request merged**